failed to run 'docker compose up'

#18 [frontend 8/8] RUN cd web && bunx --bun vite build
#18 0.817 ✘ [ERROR] JSON does not support trailing commas
#18 0.817 
#18 0.817     ../package.json:7:9:
#18 0.817       7 │     "web",
#18 0.817         ╵          ^
#18 0.817 
#18 0.832 failed to load config from /build/web/vite.config.ts
#18 0.832 error during build:
#18 0.832 Error: Build failed with 1 error:
#18 0.832 ../package.json:7:9: ERROR: JSON does not support trailing commas
#18 0.832     at failureErrorWithLog (/build/node_modules/.bun/esbuild@0.21.5/node_modules/esbuild/lib/main.js:1472:19)
#18 0.832     at <anonymous> (/build/node_modules/.bun/esbuild@0.21.5/node_modules/esbuild/lib/main.js:945:25)
#18 0.832     at buildResponseToResult (/build/node_modules/.bun/esbuild@0.21.5/node_modules/esbuild/lib/main.js:943:7)
#18 0.832     at <anonymous> (/build/node_modules/.bun/esbuild@0.21.5/node_modules/esbuild/lib/main.js:622:9)
#18 0.832     at handleIncomingPacket (/build/node_modules/.bun/esbuild@0.21.5/node_modules/esbuild/lib/main.js:677:12)
#18 0.832     at readFromStdout (/build/node_modules/.bun/esbuild@0.21.5/node_modules/esbuild/lib/main.js:600:7)
#18 0.832     at emit (node:events:95:22)
#18 0.832     at addChunk (internal:streams/readable:265:47)
#18 0.832     at readableAddChunkPushByteMode (internal:streams/readable:243:18)
#18 0.832     at handleNumberResult (internal:streams/native-readable:69:18)
#18 ERROR: process "/bin/sh -c cd web && bunx --bun vite build" did not complete successfully: exit code: 1

--- a/.dockerignore
+++ b/.dockerignore
@@ -38,7 +38,6 @@ biome.json
 .bumpversion.cfg
 .npmrc
 Makefile
-CHANGELOG.md
 CONTRIBUTING.md
 SECURITY.md
 LICENSE

--- a/Dockerfile
+++ b/Dockerfile
@@ -9,7 +9,7 @@ FROM oven/bun:1 AS frontend
 WORKDIR /build
 
 # Copy workspace config and frontend source
-COPY package.json bun.lock ./
+COPY package.json bun.lock CHANGELOG.md ./
 COPY app/ ./app/
 COPY web/ ./web/
 

